cli/src/usage.mjs

```diff
@@ -42,8 +42,8 @@ const lines = [
   { cmdInfo: "--broadcastEvent ../examples/device-onDeviceNameChanged1.event.json",       comment: "Send BroadcastEvent (method, result keys expected)" },
   { cmdInfo: "--sequence ../examples/events1.sequence.json  ",                            comment: "Send an event sequence (See examples/device-onDeviceNameChanged.sequence.json)" },
   { cmdInfo: "--session start/stop  ",                                                    comment: "Start/Stop Firebolt session recording" },
-  { cmdInfo: "--sessionOutput log|raw|mock-overrides  ",                                  comment: "Set the output format to; log: (paired time sequence of calls, responses)|raw: similiar to log but not paired with request|mock-overrides: a directory of mock overrides" },
-  { cmdInfo: "--sessionOutputPath ../examples/path  ",                                    comment: "Specifiy the session output path. Default for 'log' format will be ./output/sessions and ./output/mocks/<START_TIME> for 'mock-overrides'." },
+  { cmdInfo: "--sessionOutput log|raw|mock-overrides|live  ",                                  comment: "Set the output format to; log: (paired time sequence of calls, responses)|raw: similiar to log but not paired with request|mock-overrides: a directory of mock overrides|live: log messages as they are received in real time" },
+  { cmdInfo: "--sessionOutputPath ../examples/path  ",                                    comment: "Specifiy the session output path. Default for 'log' format will be ./output/sessions and ./output/mocks/<START_TIME> for 'mock-overrides'. Can also be a websocket url" },
```

Comment:
"Can also be a websocket url" for live only

Reply:
Updated